### PR TITLE
Scenario documentation

### PR DESCRIPTION
**Pull request by calderinisystemorph**

### CODE REVIEW

**Comment by dcolleoni** [comment]
In general, I think we need to provide more information. 
Ex. 
1. can a user add new scenarios
2. can many files be imported for the same scenario (what is our recommendation)
3. which data need to be imported for each scenario (is not enough to mention the relaxed query - add simple examples to describe what happens)
4. which recalculation occur in which cases (reimport of BE CF, reimport of SCE CF, reimport of ACT,YC,... BE or SCE)
5. other?

**Comment by calderinisystemorph** [comment]
1 can a user add new scenarios => discussed in Datamodel / Scenario

File: ifrs17/Report/ReportStorage.ipynb [hunk 1]
```diff
@@ -75,8 +90,8 @@
                 "\n    public string FunctionalCurrency { get; init; }",
                 "\n",
                 "\n    [NotAggregated]",
-                "\n    [Dimension(typeof(ProjectionConfiguration), nameof(Projection))]",
-                "\n    public string Projection { get; init; }",
+                "\n    [Dimension(typeof(int), nameof(Projection))]",
```
Comment (dcolleoni) [inline ifrs17/Report/ReportStorage.ipynb:93]: reset

**Review by dcolleoni — APPROVED** [review]

File: ifrs17/DataModel/DataStructure.ipynb [hunk 2]
```diff
@@ -617,7 +619,24 @@
...
+                "\n",
+                "\nThe only exception to this is applied to time dependency. When the user imports figures for periods $P$ and $P+1$ are imported for all scenarios and a new input for Best Estimate period $P$ is provided, only the scenarios for period $P$ are automatically updated. A manual update of period $P+1$ is then required to update the figures of $P+1$. We consider the case of restating previous periods particularly sensitive and defer to the user the resposability to ensure that all results are up to date. ",
+                "\n",
+                "\n### Dependecy with Best Estimate scenario: across periods",
+                "\n",
+                "\nWhen a stress scenario is imported for a period $P$ and a previous period $P-1$ is available, the End of Period values ($P-1$) of the Best Estimate scenario is considered as Beginning of Period ($P$) for the stressed and no-stress scenarios. This implies that:",
+                "\n1. scenarios can be occasionally not be calculated without impacting their calculation in future periods,",
```
Comment (tkukuljan) [inline ifrs17/DataModel/DataStructure.ipynb:637]: typo

**Pull request merged**